File: backend-for-frontend/citizen-data-controller-backend-for-frontend/src/main/java/br/com/lucasalbertoni/master/degree/citizen/citizendatacontrollerbackendforfrontend/entity/TrocarSenhaLogin.java
package br.com.lucasalbertoni.master.degree.citizen.citizendatacontrollerbackendforfrontend.entity;

public class TrocarSenhaLogin {
	
	private LoginToken token;
	private String senhaAtual;
	private String novaSenha;
	
	public LoginToken getToken() {
		return token;
	}
	public void setToken(LoginToken token) {
		this.token = token;
	}
	public String getSenhaAtual() {
		return senhaAtual;
	}
	public void setSenhaAtual(String senhaAtual) {
		this.senhaAtual = senhaAtual;
	}
	public String getNovaSenha() {
		return novaSenha;
	}
	public void setNovaSenha(String novaSenha) {
		this.novaSenha = novaSenha;
	}

}

File: citizendatacontroller/lib/models/trocarsenhalogin.dart
import 'package:citizendatacontroller/models/logintoken.dart';

class TrocarSenhaLogin{
  final LoginToken token;
  final String senhaAtual;
  final String novaSenha;
  TrocarSenhaLogin(this.token, this.senhaAtual, this.novaSenha);
}